rust/gui-client/build.sh

```diff
@@ -8,12 +8,12 @@ BUNDLES_DIR=../target/release/bundle/deb
 # Copy frontend dependencies
 cp node_modules/flowbite/dist/flowbite.min.js src/
 
-# Compile TypeScript
-pnpm tsc
-
 # Compile CSS
 pnpm tailwindcss -i src/input.css -o src/output.css
 
+# Compile TypeScript
+pnpm tsc && pnpm vite build
```

Comment:
I am surprised `vite` doesn't compile the typescript for you. Are you sure we need the `tsc` step?